alphagov/paas-product-pages
# Update and cleanup dependencies

## What

- regular dependency update
- cleanup of unused dependencies
- changes needed with updated dependencies

## How to test

- clone branch
- install updated dependencies (`npm install`)
- build the site (`npm run build`)
- `cd` into `out` folder and serve the page (`npx serve`)
- check page on `localhost:5000` for any issues

There are [expected warnings in CSS compilation](https://github.com/alphagov/govuk-frontend/issues/2238) coming from govuk-frontend

`npm install` will output 6 high severity vulnerabilities. 
They're slowdowns and we cannot do anything about them for now. See https://github.com/hashicorp/next-mdx-remote/issues/144
